src/build.props

```diff
@@ -20,8 +20,8 @@
     having all the relevant values in one place. This property is actually used by the PowerShell
     script that hides ("delists") the previous package versions on nuget.org.
     -->
-    <VersionPrefix>2.4.13</VersionPrefix>
-    <PreviousVersionPrefix>2.4.12</PreviousVersionPrefix>
+    <VersionPrefix>2.4.14</VersionPrefix>
+    <PreviousVersionPrefix>2.4.13</PreviousVersionPrefix>
```

Comment:
>PreviousVersionPrefix [](http://example.com/codeflow?start=5&length=21)

This comment says that the 'delist' command uses this property. Is that true? If not, let's please delete all this stuff.

If we *could* delist in future, please update this source code comment with an accurate state of the world.

Reply:
Removed the comment.